sentry/src/main/java/io/sentry/Breadcrumb.java

```diff
@@ -663,6 +663,7 @@ public void setUnknown(@Nullable Map<String, Object> unknown) {
   @Override
   @SuppressWarnings("JavaUtilDate")
   public int compareTo(@NotNull Breadcrumb o) {
+    // TODO also use nano time if equal
```

Comment:
To fix breadcrumb ordering across mutliple collections when combining scopes - so we can maintain insertion order as well as possible. If this isn't good enough we might need some atomic counter we increment for each breadcrumb.